Allow reading calendars from files (#1204)
collective/icalendar: docs/how-to/usage.rst

This section describes how to use icalendar's essential features.

Read calendar from file
'''''''''''''''''''''''

Read a calendar from a local :file:`.ics` file and view its events.

.. code:: pycon

    >>> import icalendar
    >>> from pathlib import Path
    >>> ics_path = Path("src/icalendar/tests/calendars/example.ics")
    >>> calendar = icalendar.Calendar.from_ical(ics_path)
    >>> for event in calendar.events:
    ...     print(event.get("SUMMARY"))
    New Year's Day
    Orthodox Christmas
    International Women's Day

Read calendar from URL
''''''''''''''''''''''

> Read a calendar from a URL to an :file:`.ics` file. Download the data first, then parse it.

Reviewer: This is more grammatically correct and technically accurate.

```suggestion
To read a calendar from a URL, retrieve the data first, then parse it in icalendar.
```
